lib/cli.js

```diff
@@ -23,19 +23,9 @@ summarizeAll(path.resolve(concatStatsForPath), function(entry) {
 });
 
 spinner.text = 'complete, checkout: ' + concatStatsForPath + '*.out.json';
-const cpr = require('cpr').cpr; //Back compat
 
-cpr(path.join(__dirname, '..', '/output'), concatStatsForPath, {
-  overwrite: true, //If the file exists, overwrite it
-}, (err/*, files*/) => {
-  spinner.stopAndPersist();
-  if (err) { console.error(err); return }
-  let summary = 'var SUMMARY = ' + JSON.stringify(buildOuputSummary(process.cwd() + '/concat-stats-for/'), null, 2);
```

Comment:
@stefanpenner just to make sure I did not miss something here: I removed the instances of `process.cwd() + '/concat-stats-for/'` and replaced them by `concatStatsForPath` (which comes from `process.argv[2]`, and usually is `./concat-stats-for`), because AFAICT there should be just one path used, right?